projects/packages/wordads/tools/webpack.dashboard.config.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+const jetpackWebpackConfig = require( '@automattic/jetpack-webpack-config/webpack' );
+const path = require( 'path' );
+
+module.exports = {
+	mode: jetpackWebpackConfig.mode,
+	devtool: jetpackWebpackConfig.isDevelopment ? 'source-map' : false,
+	entry: {
+		'jp-wordads-dashboard': path.join( __dirname, '../src/dashboard/index.jsx' ),
+	},
+	output: {
+		...jetpackWebpackConfig.output,
+		path: path.join( __dirname, '../build/dashboard' ),
+	},
+	optimization: {
+		...jetpackWebpackConfig.optimization,
+		splitChunks: {
+			cacheGroups: {
+				vendors: false,
+			},
+		},
+	},
+	resolve: {
+		...jetpackWebpackConfig.resolve,
+		alias: {
+			...jetpackWebpackConfig.resolve.alias,
+			fs: false,
+		},
+		modules: [ path.resolve( __dirname, '../src/dashboard' ), 'node_modules' ],
+	},
+	plugins: [
+		...jetpackWebpackConfig.StandardPlugins( {
+			DependencyExtractionPlugin: {
+				injectPolyfill: true,
+			},
+		} ),
+	],
+	externals: {
+		jetpackConfig: JSON.stringify( {
+			consumer_slug: 'jetpack-wordads-package',
+		} ),
+	},
+	module: {
+		strictExportPresence: true,
+		rules: [
+			// Transpile JavaScript
+			jetpackWebpackConfig.TranspileRule( {
+				exclude: /node_modules\//,
+			} ),
+
+			//  Transpile @automattic/jetpack-* in node_modules too.
+			jetpackWebpackConfig.TranspileRule( {
+				includeNodeModules: [ '@automattic/jetpack-' ],
+			} ),
+
+			// Handle CSS.
+			jetpackWebpackConfig.CssRule( {
+				extensions: [ 'css', 'sass', 'scss' ],
+				extraLoaders: [
+					{
+						loader: 'postcss-loader',
```

Comment:
We discovered an issue here, the `postcss-custom-properties.preserve` is set to false, which leads to css variables are not compiled into the target file. Currently we removed `postcss-loader`.

Reply:
Fixed, thanks